ister.py

```diff
@@ -413,8 +413,6 @@ def copy_os(args, template, target_dir):
     """Wrapper for running install command
     """
     LOG.info("Starting swupd. May take several minutes")
-    if not args.format:
```

Comment:
Similarly to content and version urls, I think we can just remove the grabbing the format from the filesystem entirely.

Reply:
Ah, ok.  I'll take this out in the "Remove legacy definitions" commit